components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "Install Chef Infra Server With Automate"
+description = "Use Automate to install Chef Infra Server"
+date = 2020-02-11T14:24:00-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 60
++++
+
+## Overview
+Use Chef Automate to install Chef Infra Server.
+
+## System requirements and prerequisites
+### Hardware
+See the [hardware requirements for Chef Infra Server](https://docs.chef.io/server_overview.html) for guidance on memory and number of CPUs for a standalone Chef Infra Server installation.
+
+For a single-host installation that contains Chef Infra Server and Chef Automate, start
+with
```

Comment:
Suggestions on disk/memory for Automate + Infra Server? Aside from the disk reqs, these are just the reqs for Automate so far.